entries/QUnit.module.xml

```diff
@@ -15,16 +15,28 @@
 				<desc>Runs after each test</desc>
 			</property>
 		</argument>
+		<argument name="contained_suite" optional="true">
```

Comment:
I used suite to define the grouped tests under that module. 

I can change to `contained_tests` or even `grouped_tests`.